### PR TITLE
7285   qgis training manual   project snapping options

### PR DESCRIPTION
**Pull request by tlh22**
Deals with #7285. A few points:

- The form has been changed and instructions changed/added
- Spaces have been added after images (throughout this document) - although this generated warnings in the make. (Is this something that is handled with a style?)
- Substitutions have been added at the end of the file

### CODE REVIEW

File: docs/training_manual/forestry/stands_digitazing.rst [hunk 1]
```diff
@@ -40,6 +40,7 @@ original scanned map:
 
 .. figure:: img/gimp_map.png
    :align: center
+|
```
Comment (DelazJ) [inline docs/training_manual/forestry/stands_digitazing.rst:43]:
```suggestion

|
```
Missing blank line

File: docs/training_manual/forestry/stands_digitazing.rst [hunk 2]
```diff
@@ -72,7 +74,8 @@ layer and then save it as separate image file:
 
 .. figure:: img/saving_green_px.png
    :align: center
-   
+!
```
Comment (DelazJ) [inline docs/training_manual/forestry/stands_digitazing.rst:77]:
```suggestion

|
```

File: docs/training_manual/forestry/stands_digitazing.rst [hunk 3]
```diff
@@ -356,7 +370,11 @@ might be more interested in performing analysis in a dataset more up to date.
 The topic of the next lesson will be the creation of forest stands using current
 aerial photos and the addition of some relevant information to your dataset.
 
-
+.. |topologicalEditing| image:: /static/common/mIconTopologicalEditing.png
```
Comment (DelazJ) [inline docs/training_manual/forestry/stands_digitazing.rst:373]: put these in the ordered list from line 384 (I can't see why the script didn't handle this)

Comment (tlh22) [reply docs/training_manual/forestry/stands_digitazing.rst]: Changed in PR

File: docs/training_manual/forestry/stands_digitazing.rst [hunk 4]
```diff
@@ -167,24 +179,28 @@ have already been digitized:
...
-* Check the :guilabel:`Avoid Int.` box for the :kbd:`forest_stands` layer.
-* Check :guilabel:`Enable topological editing`.
-* Click :guilabel:`Apply`.
+* Go to :menuselection:`Project --> Snapping options...`
+* Check the :kbd:`green_centroids` and :kbd:`forest_stands` layers
+* Set :guilabel:`Type` for each layer to :kbd:`Vertex`
+* Set  :guilabel:`Tolerance` for each layer to :kbd:`10 pixels`
+* Check the :guilabel:`Avoid Overlap` box for the :kbd:`forest_stands` layer
+* Check |topologicalEditing| :sup:`Topological editing`
+* Choose |avoidIntersectionsLayers| :sup:`Follow Advanced Configuration`
```
Comment (DelazJ) [inline docs/training_manual/forestry/stands_digitazing.rst:188]: sup is for tooltip and guilabel for "direct" label
```suggestion
* Press |topologicalEditing| :guilabel:`Topological editing`
* Choose |avoidIntersectionsLayers| :guilabel:`Follow Advanced Configuration`
```

**Comment by DelazJ** [comment]
Thanks @tlh22 
If you want to go further on this... There are some sphinx "rules" misuse in the file:
* :kbd: is meant for keyboard commands while it is used for various texts
* \* should not be used to indicate steps; we should instead use #.

**Pull request merged**